packages/unmock-core/src/generator.ts

```diff
@@ -59,14 +59,15 @@ import {
   Schema,
 } from "./service/interfaces";
 import { ServiceStore } from "./service/serviceStore";
+
 export const runnerConfiguration = {
-  seed: 0,
+  defaultSeed: 0,
   optionalsProbability: 1.0,
   minItems: 0,
   reset() {
+    this.defaultSeed = 0;
```

Comment:
If we now use `defaultSeed` as an immutable instance, why do we need to reset it here?